src/arrayMethodSort.js

```diff
@@ -3,9 +3,41 @@
 /**
  * Implement method Sort
  */
+function compareString(a, b) {
+  const compareA = String(a);
+  const compareB = String(b);
+
+  if (compareA > compareB) {
+    return 1;
+  }
+
+  if (compareA < compareB) {
+    return -1;
+  }
+
+  return 0;
+}
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = compareString) {
+    let count;
+
+    do {
+      count = false;
```

Comment:
make more meaningful names
```suggestion
      isContinue = false;
```